docs/Development Concepts/3.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Protocols Explained by example
+---
+
+![Overview of how protocols interact in Waku v2.](./overview.png)
+
+We have six nodes, A-F.
+The protocols initially mounted are indicated as such.
+The PubSub topics `pubtopic1` and `pubtopic2` is used for routing and indicates that it is subscribed to messages on that topic for relay.
+Ditto for Waku Store where it indicates that these messages are persisted on that node.
+
+1. Node A creates a WakuMessage `msg1` with a ContentTopic `contentTopic1`.
```

Comment:
```suggestion
1. Node A creates a WakuMessage `msg1` with a ContentTopic `/my-dapp/0/notifications/proto`.
```

I think it's better to provide a dummy content topic that looks like reality so a reader has a better picture of what it would look like.

Reply:
makes sense